toqito/helper/tests/test_npa_hierarchy.py

```diff
@@ -60,7 +60,7 @@ def test_cglmp_inequality(k):
     objective = cvxpy.Maximize(i_b)
     problem = cvxpy.Problem(objective, npa)
     val = problem.solve()
-    np.testing.assert_equal(np.allclose(val, 2.914, atol=1e-3), True)
+    assert abs(val - 2.914) <= 1e-3
```

Comment:
Hmm, I am wondering why this change needed to be made. I was under the impression that the `atol` parameter would check what you're checking here, namely that the difference between `val` and 2.914 is within 3 decimal places.

Reply:
I initially checked for both being equal to each other. Then, realized had to account for the tolerance value. So, used this.  

I am actually trying to undo the current style where you check for something then also make sure that something's check was true. The new changes to that line while using numpy's testing functionality would be:

```
np.testing.assert_equal(abs(val-2.914), 1e-3)
```

Reply:
I think you can also use `rtol` without the `abs()` function as this just checks for the relative tolerance. Or, the way you have it here is also fine!